pydicom/filewriter.py

```diff
@@ -243,15 +245,14 @@ def _is_multi_value(val):
 def multi_string(val):
     """Put a string together with delimiter if has more than one value"""
     if _is_multi_value(val):
-        # \ is escape chr, so "\\" gives single backslash
         return "\\".join(val)
     else:
         return val
 
 
-def write_PN(fp, data_element, padding=b' ', encoding=None):
-    if not encoding:
-        encoding = [default_encoding] * 3
+def write_PN(fp, data_element, encodings=None):
+    if not encodings:
```

Comment:
I have a vague recollection that the padding argument (not just in this function) was added somewhere along the way to address an error of some kind or to flex with some non-conformant files.  I tried looking through commit history but couldn't find it.   I'm not against removing it, but wondered if anyone else remembered this or could find the history.

Reply:
Well not much to see in the history:
```Author: Darcy Mason Date: 9/7/2008 9:59 PM  First upload of code to online repository.```

Reply:
Note that I didn't change any logic in this case - these parameters have not been used in the code, so I removed them.